processes/security_team_members.md

```diff
@@ -0,0 +1,92 @@
+# Node.js Security Team
+
+Node.js security team members are expected to keep all information that they have
+privileged access to by being on the team completely private to the team. This
+includes notifying anyone outside the team of issues that have not yet been
+disclosed publicly, including the existence of issues, expectations of upcoming
+releases, and patching of any issues other than in the process of their work as
+a member of the security team.
+
+Membership on the security teams can be requested via an issue in the TSC repo,
+and must be approved by current team members.
+
+Members of the security teams should indicate that they accept the privacy
+policies by PRing their acceptance to this file.
+
+## Team that triages security reports against node core
+
+- @bnoordhuis - **Ben Noordhuis**
+- @indutny - **Fedor Indutny**
+- @rvagg - **Rod Vagg**
+- @jasnell - **James M Snell**
+- @shigeki - **Shigeki Ohtsu**
+- @MylesBorins - **Myles Borins**
+
+List is from ["security" alias](https://github.com/nodejs/email/blob/master/iojs.org/aliases.json).
+
+## Team with access to security issues
```

Comment:
So thinking about this particular list... I think we should manage this differently. Within the nodejs-private GitHub organization, only the Triage team above should have general access to everything. When a new security vulnerability is reported, a *new* private fork repo within nodejs-private is created by a member of the triage team, and a new GitHub team *specific* to that one repo is created by the Triage team member. That specific team can pull in any other collaborator/contributor is necessary to address that one problem so long as they agree to the confidentiality requirement. Once the issue is resolved and the fix lands in nodejs/node master, the individual fix repo in nodejs-private is archived and the GitHub team is dissolved.

This accomplishes several things:

1. We have targeted and specific control over who has access to the specific security vulnerability details
2. Each security issue will have it's own issues/PR tracker
3. We don't have to maintain a list like the one below

Reply:
I agree with the generalities of what you describe above, but the current list is a statement of fact, not of how it could be. Adoption of new tooling, proposed in https://github.com/nodejs/TSC/issues/344 and not generating much interest, would cause a change in the process. For example, some of what you describe above is specific to the  tooling used, once @vdeturckheim has made more progress on #57 , it may be easy to demonstrate the process, and propose it to be used for node core - at which point these specific teams may cease to exist, but  the ***privacy policy*** will continue  to be relevant.